internal/campaigns/archive_fetcher.go

```diff
@@ -86,6 +86,10 @@ func fetchRepositoryArchive(ctx context.Context, client api.Client, repo *graphq
 		return fmt.Errorf("unable to fetch archive (HTTP %d from %s)", resp.StatusCode, req.URL.String())
 	}
 
+	if err := os.MkdirAll(filepath.Dir(dest), 0700); err != nil {
```

Comment:
I'm assuming this won't fail if the dest already exists?

Reply:
> // If path is already a directory, MkdirAll does nothing
> // and returns nil.